### PR TITLE
chore(districts): add new district descriptions

### PR DESCRIPTION
**Pull request by rhnonose**
crates a data migration to add new districts and its descriptions

### CODE REVIEW

File: apps/re/priv/repo/migrations/20190416171337_add_new_district_descriptions.exs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      O bairro é considerado um dos mais legais da cidade, responsável por lançar tendências e conhecido por seus points gastronômicos e suas boutiques.
+      Com uma localização privilegiada, o bairro Pinheiros está situado na Zona Oeste de São Paulo. Faz divisa com os bairros Jardim Paulistano, Vila Madalena, Alto de Pinheiros, Cidade Jardim e City Butantã. A região também está localizada próxima de importantes vias como as Avenidas Brigadeiro Faria Lima, Rebouças, Eusébio Matoso e Nações Unidas, além da Marginal Pinheiros.
+      Mais do que uma excelente localização, Pinheiros está repleto de soluções e boas opções para quem mora ou frequenta o bairro. São diversos Parques, como o Villa-Lobos e a praça Panamericana. No segmento da saúde, temos o Hospital das Clínicas e, na área da educação grandes universidades também estão presentes: UNIP, FMU e FESP. Para finalizar, o bairro ainda abriga os Shoppings Eldorado e Iguatemi.
+      São essas e outras características que tornam Pinheiros, uma das regiões mais cobiçadas da cidade de São Paulo.
+      """
+    }
+  ]
+  def up do
```
Comment (joaodubas) [inline apps/re/priv/repo/migrations/20190416171337_add_new_district_descriptions.exs]: Missing a blank line between `@districts` and `up/0`

Comment (rhnonose) [reply apps/re/priv/repo/migrations/20190416171337_add_new_district_descriptions.exs]: It's weird that the formatter didn't get that.
Maybe I'll open up an issue.

Comment (rhnonose) [reply apps/re/priv/repo/migrations/20190416171337_add_new_district_descriptions.exs]: elixir-lang/elixir#8960

Comment (rhnonose) [reply apps/re/priv/repo/migrations/20190416171337_add_new_district_descriptions.exs]: It's intentional, so it has to be manual :/

**Pull request merged**